V1/utils/config.py

```diff
@@ -26,18 +26,46 @@ def load_config(path_to_config = './config.json'):
     data = json.loads(data)
     return data
 
+def create_deadline(deadline_value):
+    deadline = 45.0
+    try:
+        if deadline_value < 0:
+            print("WARNING: Value for deadline should not be negative")
+        if deadline_value != 0:
+            deadline = abs(float(deadline_value))
+        else:
+            print("ERROR: Value for deadline cannot be 0. 45.0 will be set instead.")
+    except ValueError as val_err:
+        print("ERROR: Invalid value for deadline. 45.0 will be set instead.")
+        deadline = 45.0
+    return deadline
+
+def create_idle_power(value):
+    idle_power = 5
+    try:
+        if value < 0:
+            print("WARNING: Value for idle_power should not be negative")
+        if value != 0:
+            idle_power = abs(int(value))
+        else:
+            print("ERROR: Value for idle_power cannot be 0. 5 will be set instead.")
+    except ValueError as val_err:
+        print("ERROR: Invalid value for idle_power. 5 will be set instead.")
+        idle_power = 5
+    return idle_power
+
 def create_task_types(task_types_info):
     task_types = []
     task_type_names = []
-    for task_type in task_types_info:
-        id = task_type['id']
+    for id, task_type in enumerate(task_types_info):
+        #id = task_type['id']
         name = task_type['name']
-        urgency = task_type['urgency']
+        urgency = task_type['urgency'].lower()
         if urgency == 'best_effort':
             urgency = UrgencyLevel.BESTEFFORT
         elif urgency == 'urgent':
             urgency = UrgencyLevel.URGENT
-        deadline = task_type['deadline']
+        deadline = create_deadline(task_type['deadline'])
         task_types.append(TaskType(id, name, urgency,deadline))
```

Comment:
There might be missing information in task_types_info. Like a user may delete the urgency in task_type_infor by mistake.